nesta/core/scripts/docker_recipes/py36_sklearn_amzn1_image/Dockerfile

```diff
@@ -1,25 +1,55 @@
 FROM amazonlinux:1
 
+RUN du -cxh --threshold=5M --max-depth=3 /
+
+RUN mkdir /root/.conda
+ENV PYTHONDONTWRITEBYTECODE=true
+ENV PYTHONIOENCODING=utf8
+
 # Install common dependencies
 RUN rm -rf /var/cache/yum/x86_64/latest
 RUN yum update -y
-RUN yum install sudo -y
+#RUN yum install sudo -y
```

Comment:
Should these sudo and zip lines be removed if not needed? Otherwise, a comment about when they may be needed to be uncommented would be useful.